packages/docs/docs/medications/formulary.md

```diff
@@ -0,0 +1,298 @@
+    "coding": [
+      {
+        "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+        "code": "569998"
+      },
+      {
+        "system": "http://hl7.org/fhir/sid/ndc",
+        "code": "5058049501"
+      }
+    ]
+  },
+  "drugCharacteristic": [
+    {
+      "type": {
+        "text": "Color",
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/medicationknowledge-characteristic",
+            "code": "color"
+          }
+        ]
+      },
+      "valueString": "white"
+    },
+    {
+      "type": {
+        "text": "Imprint Code",
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/medicationknowledge-characteristic",
+            "code": "imprintcd"
+          }
+        ]
+      },
+      "valueString": "Tylenol"
+    }
+  ],
+  "amount": {
+    "value": 100,
+    "unit": "{tbl}"
+  },
+  "packaging": {
+    "type": {
+      "text": "Bottle",
+      "coding": [
+        {
+          "system": "http://terminology.hl7.org/CodeSystem/medicationknowledge-package-type",
+          "code": "bot",
+          "display": "Bottle"
+        }
+      ]
+    },
+    "quantity": {
+      "value": 100,
+      "unit": "{tbl}"
+    }
+  },
+  "doseForm": {
+    "text": "tablet"
+  },
+  "intendedRoute": [
+    {
+      "text": "oral"
+    }
+  ]
+  //...
+}
+```
+
+</details>
+
+## Drug Classifications and Regulations
+
+The `MedicationKnowledge.productType` field can be used to categorize the drug within the formulary, potentially along multiple dimensions. This field can also be used to describe:
+
+- Whether a drug is generic or branded
+- Whether the drug is prescribable
+- Whether the drug is dispensable
+- Whether the drug requires a prescription (see: [Legal status of Supply](https://build.fhir.org/valueset-legal-status-of-supply.html))
+
+The `MedicationKnowledge.regulatory` contains crucial regulatory information related to drug dispensing, such as maximum permissible units and substitution regulations. In particular, `MedicationKnowledge.regulatory.schedule.schedule` is used for specifying the regulatory schedule for controlled substances.
+
+Refer to the [HL7 Controlled Substances Schedule](https://terminology.hl7.org/ValueSet-v2-0477.html) for an example valueset for substances subject to the U.S. Controlled Substances Act (CSA).
+
+## Compounded Medications
+
+The `MedicationKnowledge.ingredients` field is used for listing ingredients of compounded medications, with RxNorm as the preferred code system for reach ingredient.
+
+The `MedicationKnowledge.amount` field indicates the total amount of the compound to be dispensed.
+
+For each ingredient, the `strength` is stored as a ratio of the total volume (e.g. 1mg / 100mg). The field `ingredient.active` indicates which of the listed ingredients are active.
+
+```ts
+{
+"resourceType": "MedicationKnowledge",
+// Dispense 200g total
+"amount": {
+  "value": 200,
+  "unit": "g"
```

Comment:
Should this be `mg`?

Reply:
Will update the text above to indicate `g`. 

Grams is actually the right amount for this example